metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java

```diff
@@ -291,8 +291,17 @@ protected boolean isShutdownExecutorOnStop() {
         return shutdownExecutorOnStop;
     }
 
-    protected Set<MetricAttribute> getDisabledMetricAttributes() {
```

Comment:
Please, don't remove the `getDisabledMetricAttributes` method. All changes must be backward-compatible.